src/overlay/OverlayManagerImpl.cpp

```diff
@@ -201,10 +209,33 @@ OverlayManagerImpl::PeersList::acceptAuthenticatedPeer(Peer::pointer peer)
 
     CLOG(INFO, "Overlay") << "Non preferred " << mDirectionString
                           << " authenticated peer " << peer->toString()
-                          << " rejected because all available slots are taken.";
+                          << " rejected because all available slots are taken."
+                          << " @" << mOverlayManager.mApp.getConfig().PEER_PORT;
     CLOG(INFO, "Overlay")
         << "If you wish to allow for more " << mDirectionString
         << " connections, please update your configuration file";
+
+    if (Logging::logTrace("Overlay"))
+    {
+        CLOG(TRACE, "Overlay") << fmt::format(
+            "limit: {}, pending: {}, authenticated: {}", mMaxAuthenticatedCount,
+            mPending.size(), mAuthenticated.size());
+        std::stringstream pending, authenticated;
+        for (auto p : mPending)
+        {
+            pending << p->toString();
```

Comment:
Do you want to add the port here as well?

Reply:
no: port is unrelated to `p` and there is already context from the `INFO` entry above